### ISSUE TITLE
[Entry Request] Python: Iterators

### ISSUE THREAD

**Issue by sonnynomnom** [Codecademy/docs#287]
## Description

Write an "Iterators" entry in the Python topic.

General topic: https://www.codecademy.com/resources/docs/python

## Type of Entry Request

- One or more new entries/terms requested.

## Checklist

- [x] The issue is appropriately titled.
- [x] Description is clear and includes supporting text, files, etc.
- [x] A similar issue hasn't already been created.
- [x] No more than 3 entries requested.

## For Maintainers

- [x] Labels are added
- [x] Assignee is added

**Comment by sammy8825** [comment]
I would like to work on this issue

**Comment by sonnynomnom** [comment]
Hi Sammy, I just assigned you another issue. So maybe wait on this one to see anyone else wants to take it first.

**Comment by sammy8825** [comment]
Okk

**Comment by daksh10110** [comment]
Hey, can you assign this issue to me, I would like to work on it

**Comment by sonnynomnom** [comment]
Hi @daksh10110 I just assigned you another issue!

**Comment by DeepakReddyVelagala** [comment]
thanks, I searched for hacktoberfest and found this

**Comment by EugeneGohh** [comment]
> could you please direct me on how to proceed

Hi there, there is a headstart here on the [README](https://github.com/Codecademy/docs). Please go through the README and "[How do I contribute?](https://github.com/Codecademy/docs#how-do-i-contribute)" section.
